### PR TITLE
fix(ban user): dont close overlay by state update

### PR DESCRIPTION
**Pull request by RunzelRosinchen**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: src/components/groupChat/GroupChatInfo.tsx [hunk 1]
```diff
@@ -65,9 +65,11 @@ export const GroupChatInfo = () => {
 	const [overlayItem, setOverlayItem] = useState<OverlayItem>(null);
 	const [overlayActive, setOverlayActive] = useState(false);
 	const [redirectToSessionsList, setRedirectToSessionsList] = useState(false);
+	const [isUserBanOverlayOpen, setIsUserBanOverlayOpen] =
+		useState<boolean>(false);
 	const [isRequestInProgress, setIsRequestInProgress] = useState(false);
 	const [bannedUsers, setBannedUsers] = useState<string[]>([]);
-	const [isV2GroupChat, setIsV2GroupChat] = useState<boolean>(false);
+	const [isV2GroupChat, setIsV2GroupChat] = useState<boolean>(true);
```
Comment (web-mi) [inline src/components/groupChat/GroupChatInfo.tsx]: War das zum testen?

Comment (RunzelRosinchen) [reply src/components/groupChat/GroupChatInfo.tsx]: oh yes 🙈

Comment (RunzelRosinchen) [inline src/components/groupChat/GroupChatInfo.tsx]:
```suggestion
	const [isV2GroupChat, setIsV2GroupChat] = useState<boolean>(false);
```

**Review by web-mi — DISMISSED** [review]

File: src/components/banUser/BanUser.tsx [hunk 2]
```diff
@@ -14,15 +14,41 @@ interface BanUserProps {
...
+			if (handleUserBan) handleUserBan(userName);
+		});
+	};
+
+	return (
+		<>
+			<button className="banUser" onClick={banUser}>
+				{translate('banUser.ban.trigger')}
+			</button>
+		</>
```
Comment (ghost) [inline src/components/banUser/BanUser.tsx]:
```suggestion
		<button className="banUser" onClick={banUser}>
			{translate('banUser.ban.trigger')}
		</button>
```

I believe there’s no need for this wrapper here :)

**Review by SheepFromHeaven — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**